components/HelpScreen.tsx

```diff
@@ -31,39 +57,102 @@ export const HelpScreen: React.FC<HelpScreenProps> = props => {
         <ScrollView>
           <Column fill padding="10" align="space-between">
             <Text margin="7" style={Theme.TextStyles.header}>
-              {t('whatIsDigitalCredential?')}
+              {t('whatIsAnId?')}
+            </Text>
+            <Text style={Theme.TextStyles.helpDetails}>{t('detail-10')}</Text>
+            <Text margin="7" style={Theme.TextStyles.header}>
```

Comment:
can we add the margin inside the styles itself

Reply:
header in the TextStyles itself is an object from Themes. Here we can't. I suggest adding a margin in the header object, leading to changes for other heading in the app.

Reply:
Then Lets add this style as separate style in themes and include that style in the array.

Reply:
Done @vijay151096